templates/default/observablehq.config.js.tmpl

```diff
@@ -16,6 +16,9 @@ export default {
   //   }
   // ],
 
+  // Links to add to the head of the page, e.g. for a favicon:
```

Comment:
```suggestion
  // Content to add to the head of the page, e.g. for a favicon:
```

Not limited to links here (e.g., you could have an analytics script or meta tags).